docs/api.md

```diff
@@ -298,44 +240,6 @@ hook_config = DebuggerHookConfig(
 | ```coll.add_module_tensors(module, inputs=False, outputs=True)```  | **(PyTorch only)** Takes an instance of a PyTorch module and logs input/output tensors for that module. By default, only outputs are saved. |
 | ```coll.add_block_tensors(block, inputs=False, outputs=True)``` | **(MXNet only)** Takes an instance of a Gluon block,and logs input/output tensors for that module. By default, only outputs are saved. |
 
-### Configuring Collection using SageMaker Python SDK
```

Comment:
What is the alternative to this section?

Reply:
I moved it to the SageMaker pysdk debugger.py docs: https://sagemaker.readthedocs.io/en/stable/api/training/debugger.html#sagemaker.debugger.CollectionConfig